index.js

```diff
@@ -41,7 +41,7 @@ export default function(time, now) {
         cycle = 'second';
     }
 
-    const v = Math.floor(amt);
+    const v = Math.round(amt);
```

Comment:
Thanks to this change, now CI passes too https://github.com/fregante/twas/actions/runs/4121819812/jobs/7117884888